### PR TITLE
metabake

### PR DESCRIPTION
**Pull request by cekvenich**
adding the metabake static gen project, featuring Pug.

### CODE REVIEW

**Comment by cekvenich** [comment]
Not sure why it is failing.

**Comment by jnrdt** [comment]
An issue is open

https://github.com/netlify/staticgen/issues/400

File: content/projects/metabake.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+---
+title: MetaBake.org
+repo: metabake/_mBake
+homepage: https://metabake.org
+language:
+  - Pug
+license:
+  - GNU GPL v3.0
```
Comment (erquhart) [inline content/projects/metabake.md]: Please change to "GPL v3.0" (that's just what we're using for that license in the list).

File: content/projects/metabake.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+templates:
+  - Pug (w/ RIOTjs)
```
Comment (erquhart) [inline content/projects/metabake.md]: Please change to "Pug".

Comment (cekvenich) [reply content/projects/metabake.md]: I'll have to do a new PR.

Comment (erquhart) [reply content/projects/metabake.md]: You can just push an additional commit with "Pug (w/ RIOTjs)" changed to "Pug".

Comment (cekvenich) [reply content/projects/metabake.md]: I made those changes - still no joy. 
"5:51:04 PM:   SyntaxError: Unexpected end of JSON input
5:51:04 PM:   - JSON.parse
5:51:05 PM:   - fetch-archive.js:87 _callee5$"

I'd love to get on static gen home page.

File: content/projects/metabake.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
+---
+title: Metabake.org
+repo: metabake/_mBake
+homepage: http://www.metabake.org
+language: Pug
```
Comment (erquhart) [inline content/projects/metabake.md]: Sorry, missed this: Pug is the template language, but the language this generator is written in should be `JavaScript`.

**Comment by cekvenich** [comment]
Thank you!

**Pull request merged**